lib/EnzymeCore/src/EnzymeCore.jl

```diff
@@ -206,6 +207,24 @@ end
 @inline batch_size(::BatchMixedDuplicated{T,N}) where {T,N} = N
 @inline batch_size(::Type{BatchMixedDuplicated{T,N}}) where {T,N} = N
 
+"""
+    Seed(dy)
+
+Wrapper for a single adjoint to the return value in reverse mode.
+"""
```

Comment:
something that's ambiguous to me here is whether this enforces active vs duplicated for a return

Reply:
In the current state of the PR, the return's activity is specified elsewhere. Should we change that?